src/app_service/service/community/dto/community.nim

```diff
@@ -334,35 +331,32 @@ proc parseCommunities*(response: RpcResponse[JsonNode]): seq[CommunityDto] =
   result = map(response.result.getElems(),
     proc(x: JsonNode): CommunityDto = x.toCommunityDto())
 
-proc parseCuratedCommunities*(response: RpcResponse[JsonNode]): seq[CuratedCommunity] =
+proc parseKnownCuratedCommunities(jsonCommunities: JsonNode): seq[CommunityDto] =
+  for communityJson in jsonCommunities:
+    var community = communityJson.toCommunityDto()
+    community.available = true
+    community.isAlsoCuratedCommunity = true
+    result.add(community)
+
+proc parseUnknownCuratedCommunities(jsonCommunities: JsonNode): seq[CommunityDto] =
+  for communityId in jsonCommunities.items():
+    var community = CommunityDto()
+    community.id = communityId.getStr
+    community.available = false
+    community.isAlsoCuratedCommunity = true
```

Comment:
Yeah I used `isCuratedCommunity` at first, but then I didn't know if it might be confusing that it's only a curated community. I'm removing `also` if people prefer that.

Reply:
Let's have a feedback from Patryk ))

Reply:
TBH I don't have a strong opinion on that. Maybe `isListedInDirectory`? it sounds to be the least confusing to me, as it does not imply a type of community but rather a property. Then we could have `isFeaturedInDirectory` when featured communities are implemented.

Reply:
I think just `curated` would also be an option no? We also have `spectated` and `joined` respectively.